ibc-core/ics02-client/types/src/status.rs

```diff
@@ -38,6 +40,14 @@ impl Status {
     pub fn is_expired(&self) -> bool {
         *self == Status::Expired
     }
+
+    /// Checks whether the status is active; returns `Err` if not.
+    pub fn verify_active(&self) -> Result<(), ClientError> {
+        match self.clone() {
+            Self::Active => Ok(()),
```

Comment:
We can take the `self` here, so not getting it cloned on each of the `verify_is_active` calls.

```rust
pub fn verify_is_active(self) -> Result<(), ClientError> {
    match self {
        Self::Active => Ok(()),
        status => Err(ClientError::ClientNotActive { status }),
    }
}
```